pandas/core/reshape/concat.py

```diff
@@ -200,7 +200,7 @@ def concat(
 
         .. versionchanged:: 1.0.0
 
-           Changed to not sort by default.
+            Changed to not sort by default.
```

Comment:
here it looks like an extra space has crept in - could you remove it please? This line shouldn't change (likewise for many similar lines below)